### PR TITLE
(FACT-2233) Add AIX partitons fact

### PR DESCRIPTION
**Pull request by oanatmaria**

### CODE REVIEW

File: lib/resolvers/aix/partitions.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+            lsl_content.flatten!.select! { |elem| elem }.map! { |elem| elem.delete("\s") }
+
+            Hash[*lsl_content]
+          end
+
+          def compute_size(info_hash)
+            physical_partitions = info_hash['PPs'].to_i
+            size_physical_partition = info_hash['PPSIZE']
+            exp = size_physical_partition[/mega/] ? 1024**2 : 1024**3
```
Comment (Filipovici-Andrei) [inline lib/resolvers/aix/partitions.rb:76]: I think these values should be defined in constants

Comment (oanatmaria) [reply lib/resolvers/aix/partitions.rb]: Done

File: lib/resolvers/aix/utils/odm_query.rb [hunk 2]
```diff
@@ -27,9 +27,9 @@ def like(field, value)
     def execute
       result = nil
       REPOS.each do |repo|
-        break if result
+        break if result && !result.empty?
 
-        result, _s = Open3.capture2("#{query} #{repo}")
+        result, _stderr, _s = Open3.capture3("#{query} #{repo}")
```
Comment (sebastian-miclea) [inline lib/resolvers/aix/utils/odm_query.rb:32]: Why use capture3 if _stderr_ is not used?

Comment (oanatmaria) [reply lib/resolvers/aix/utils/odm_query.rb:32]: If stderr is not captured, it will be displayed when the command is executed

**Review by Filipovici-Andrei — APPROVED** [review]

**Pull request merged**